data/io.github.flattool.Warehouse.metainfo.xml.in

```diff
@@ -64,6 +64,14 @@
     </screenshot>
   </screenshots>
   <releases>
+      <release version="1.6.5" date="2024-9-18" timestamp="1720282600">
```

Comment:
Not sure if you were using a tool to manage the releases, but I haven't changed the timestamp from the previous release, just the date.